### PR TITLE
feat: Adds getLatestCheckpointForAllChains

### PR DESCRIPTION
**Pull request by mw2000**
This allows users to query across multiple chains in one call without having to call getLatestCheckpoint(chainId) multiple times

### CODE REVIEW

File: docs/WitnessClient.md [hunk 1]
```diff
@@ -79,7 +80,7 @@ Constructs an instance of `WitnessClient`.
 
 #### Defined in
 
-[src/WitnessClient.ts:88](https://github.com/WitnessCo/client/blob/c286679/src/WitnessClient.ts#L88)
+[src/WitnessClient.ts:88](https://github.com/mw2000/witness-client/blob/d898ca0/src/WitnessClient.ts#L88)
```
Comment (sina-wit) [inline docs/WitnessClient.md:83]: Wrong link

File: docs/WitnessClient.md [hunk 2]
```diff
@@ -91,7 +92,7 @@ The authentication token to use for requests.
 
 #### Defined in
 
-[src/WitnessClient.ts:89](https://github.com/WitnessCo/client/blob/c286679/src/WitnessClient.ts#L89)
+[src/WitnessClient.ts:89](https://github.com/mw2000/witness-client/blob/d898ca0/src/WitnessClient.ts#L89)
```
Comment (sina-wit) [inline docs/WitnessClient.md:95]: wrong link

**Review by sina-wit — APPROVED** [review]

File: docs/WitnessClient.md (hunk 1 above)
Comment (mw2000) [reply docs/WitnessClient.md:83]: Good catch, fixing!

File: docs/WitnessClient.md (hunk 2 above)
Comment (mw2000) [reply docs/WitnessClient.md]: Same as above

**Comment by mw2000** [comment]
@sina-wit Manual tests finished, this should be good to merge.

**Review by sina-wit — APPROVED** [review]

**Pull request merged**